hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -35,6 +35,7 @@
   // metadata dir but in future we may support multiple for redundancy or
   // performance.
   public static final String OZONE_SCM_DB_DIRS = "ozone.scm.db.dirs";
+  public static final String OZONE_SCM_DB_WAL_DIR = "ozone.scm.db.wal.dir";
 
```

Comment:
Do we know that WAL performance is being compromised today ? This adds a new config value, but are we sure about the benefit ? In other words, are we sure that ozone is being bottlenecked by RocksDB?